policies-procedures-and-docs/payments/invoicing.md

```diff
@@ -0,0 +1,31 @@
+# Invoicing OLS
+
+When invoicing OLS for a pre-agreed fee, please make sure to send the invoice to `financemanagement at openlifesci dot org`. These invoices are usually paid once a week. 
```

Comment:
Can you update the email address? The best would be to have them a Jekyll variables

Reply:
Done. 👍 
I added a new variable called `finance_email`.